docs/sample_config.yaml

```diff
@@ -441,6 +441,20 @@ retention:
 #
 #request_token_inhibit_3pid_errors: true
 
+# Controls whether locally-created rooms should be end-to-end encrypted by
+# default.
+#
+# Possible options are "all", "invite", and "off". They are defined as:
+#
+# * "all": any locally-created room
+# * "invite": any locally-created room that requires an invite to join
```

Comment:
this doesn't seem to match the implementation: the server chooses a profile based on `visibility`, not `join rules` ?

Reply:
I suppose we should just change this to preset?

Reply:
yeah, I think we should.